debugging-r.Rmd

```diff
@@ -1,5 +1,3 @@
-# (PART) All is fail {-}
-
```

Comment:
Are we losing the part structure? Or am I going to find something below that brings it back?

Reply:
The structure is preserved, just setting the options from the `_quarto.yml` instead.